Source/DafnyLanguageServer/Workspace/ChangeProcessors/Relocator.cs

```diff
@@ -82,6 +86,10 @@ CancellationToken cancellationToken
         });
       }
 
+      public Range MigrateRange(Range range) {
```

Comment:
Question: MigrateRange always returns a range, even if the original range was deleted. Won't this cause a problem?